### PR TITLE
Hit the learderboard api

### PR DESCRIPTION
**Pull request by Shedrack-Sunday**
## Changes made on this feature branch include:

- [✔️ ] Send and receive data from an API.

- [ ✔️ ] Use API documentation.

- [✔️  ] Understand and use JSON.

- [ ✔️ ] Make JavaScript code asynchronous.

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
Hi @Shedrack-Sunday, 

Very nice project. I love the way you have separated your code into modules and assigned specific tasks to specific modules 👏🏿 . However, there are a few changes to be made kindly check below ⬇️

### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️

 ## Required Changes ♻️

- [ ]  It will be great to fix this error. When I try posting to the endpoint, I get a `400` error. This means that the app is sending a bad request to the server. Kindly fix this error and submit your project for another review. ⬇️ 

no post


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Shedrack-Sunday** [comment]
> Hi @Shedrack-Sunday,
> 
> Very nice project. I love the way you have separated your code into modules and assigned specific tasks to specific modules 👏🏿 . However, there are a few changes to be made kindly check below ⬇️
> 
> ### Highlights!
> [ ] Good feature branch name ✔️ [ ] Descriptive pull request ✔️
> 
> ## Required Changes ♻️
> * [ ]   It will be great to fix this error. When I try posting to the endpoint, I get a `400` error. This means that the app is sending a bad request to the server. Kindly fix this error and submit your project for another review. ⬇️
> 
> no post
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hello @peterrobert !!
Thanks for the prompt reply. I've had a challnage fixing this, and I'll be glad if you can give me a hint on what went wrong. Thanks

File: src/index.js [hunk 1]
```diff
@@ -4,28 +4,90 @@ import './style.css';
...
+// Getting THE API from the endpoint
+const getAPIs = async () => {
+  const response = await fetch(`
+    https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${keyGame}/scores`);
+  const data = await response.json();
+  objPlayers.players = data.result;
+  objPlayers.playerDisplay();
+  return data.result;
+};
+
```
Comment (TedLivist) [inline src/index.js:51]: - [ ] While I don't receive the same error as the previous reviewer when adding a score I am now unable to get the scores added. I added different scores but they are not fetched and displayed when I click the refresh button or reload the page. Kindly fix this.

- [ ] **[OPTIONAL]** I would also recommend you fix your app. It displays a lot of unnecessary `undefined` name.

  rdun

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Shedrack-Sunday,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cwaku — APPROVED** [review]
Hi @Shedrack-Sunday 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**